webdriver-spec.html

```diff
@@ -576,10 +576,10 @@
 
   <section>
     <h3>Handling Errors</h3>
-    <p>Errors are represented in the WebDriver protocol with a HTTP response with a HTTP status in the 4xx or 5xx range and a JSON body containing details of the error. This JSON body has two fields; <code>error</code>, containing a string indicating the error type and <code>error</code> containing an implementation-defined string containing a human readable description of the kind of error that occured.</p>
+    <p>Errors are represented in the WebDriver protocol with a HTTP response with a HTTP status in the 4xx or 5xx range and a JSON body containing details of the error. This JSON body has two fields: <code>status</code>, containing a string indicating the error type and <code>error</code> containing an implementation-defined string containing a human readable description of the kind of error that occured.</p>
```

Comment:
This should be `error` for the string indicating the error type and `message` for the string with the human readable description.

(See the steps under **sending an error**.)